modules/claims_api/app/sidekiq/claims_api/evidence_waiver_builder_job.rb

```diff
@@ -6,6 +6,7 @@
 module ClaimsApi
   class EvidenceWaiverBuilderJob < ClaimsApi::ServiceBase
     include ClaimsApi::EwsVBMSSidekiq
+    sidekiq_options expires_in: 48.hours, retry: true
```

Comment:
Would [`retry_for: 48.hours`](https://github.com/sidekiq/sidekiq/wiki/Error-Handling#:~:text=As%20of%20Sidekiq%207.1.3%2C%20you%20can%20retry%20for%20a%20period%20of%20time%3A) work here? e.g., https://github.com/department-of-veterans-affairs/vets-api/blob/387b6c3de195170969d1fef5d5e843a500e4192b/app/sidekiq/lighthouse/poll_form526_pdf.rb#L29

Reply:
@tycol7  what is the preference for `retry_for` vs. `expires_in`?  Just curious more then anything else, as long as either works I do not care really, although `expires_in` does appear to be utilized more in the repo overall (neither seems to be in there much though) then `retry_for`.  `retry_for` does kind of make me want to expect a number of retries tbh, but I'm not really tied to either as a preference.

Reply:
Also kind of  leaning towards it since we do use this in some of the 526 tests, again, not married to it at all, but I would go with this unless there is a clear reason to alter it.

Reply:
No preference other than it's shorter :)